package/yast2-users.changes

```diff
@@ -6,6 +12,12 @@ Mon Nov 25 15:37:16 UTC 2019 - Imobach Gonzalez Sosa <igonzalezsosa@suse.com>
   - Write login.defs configuration to /etc/login.defs.d/.
 - 4.2.6
 
+-------------------------------------------------------------------
+Thu Nov 21 16:15:00 UTC 2019 - Ludwig Nussel <lnussel@suse.de>
+
+- Disable block device selector in WSL (boo#1154238)
+- 4.2.6
```

Comment:
This entry looks wrong, probably a wrongly resolved conflict...

Reply:
indeed